adapted-articles/J/JohntheBaptist.xml

```diff
@@ -2,43 +2,43 @@
-<p>The unusual dress of John the Baptist—“clothing made of camel’s hair, with a leather belt around his waist” (<a href="?bref=Mark.1.6">Mk 1:6</a>, <span class="sc">niv</span>)—may have suggested to his audience an association with Elijah in particular (<a href="?bref=2Kgs.1.8">2 Kgs 1:8</a>) or with the prophets in general (<a href="?bref=Zech.13.4">Zec 13:4</a>). His diet, “locusts and wild honey” (<a href="?bref=Mark.1.6">Mk 1:6</a>), was Levitically clean, reflecting one who lived off the desert (such food was also eaten at Qumran) and formed part of the broader asceticism practiced by John and his disciples (<a href="?bref=Matt.9.14">Mt 9:14</a>; <a href="?bref=Matt.11.18">11:18</a>).</p>
+<p class="fl">Mark’s Gospel begins with an account of John the Baptist’s appearance: “John the Baptist . . . lived in the wilderness and was preaching that people should be baptized to show that they had turned from their sins and turned to God to be forgiven” (<a href="?bref=Mark.1.4">Mk 1:4</a>). A rich OT background lies behind John’s association with the wilderness, in this case the wilderness of Judea. It was in the wilderness that God revealed himself to Moses (<a href="?bref=Exod.3.1-22">Ex 3</a>), gave the law, and entered into the covenant with Israel (ch <a href="?bref=Exod.19.1-25">19</a>). It was also the site of refuge for David (<a href="?bref=1Sam.23.1-26.25">1 Sm 23–26</a>; <a href="?bref=Ps.63.1-11">Ps 63</a>) and Elijah (<a href="?bref=1Kgs.19.1-21">1 Kgs 19</a>), and in this light became the anticipated site of God’s future deliverance (<a href="?bref=Isa.40.3-5">Is 40:3-5</a>; <a href="?bref=Ezek.47.1-12">Ez 47:1-12</a>; <a href="?bref=Hos.2.14-15">Hos 2:14-15</a>).</p>
+<p>The unusual dress of John the Baptist—“clothing made of camel’s hair, with a leather belt around his waist” (<a href="?bref=Mark.1.6">Mk 1:6</a>)—may have suggested to his audience an association with Elijah in particular (<a href="?bref=2Kgs.1.8">2 Kgs 1:8</a>) or with the prophets in general (<a href="?bref=Zech.13.4">Zec 13:4</a>). His diet, “locusts and wild honey” (<a href="?bref=Mark.1.6">Mk 1:6</a>), was clean according to the Leviticus, therefore reflecting one who lived off the desert (such food was also eaten at Qumran) and formed part of the broader asceticism practiced by John and his disciples (<a href="?bref=Matt.9.14">Mt 9:14</a>; <a href="?bref=Matt.11.18">11:18</a>).</p>
```

Comment:
"according to Leviticus" or "according to the Levites"

Reply:
> "according to Leviticus" or "according to the Levites"

That is an excellent question. Since the word “Levitically” has a capital letter, I assumed it meant according to the Leviticus. Also, if it were according to an “oral law,” wouldn’t the author have said so more clearly such as “according to the Scribes” or “according to the Sadducees.”

Either way, I’m happy to go with your recommendation. We could also leave the decision up to the translator.

Let me know.

Thanks.